src/test/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1Test.java

```diff
@@ -81,7 +80,11 @@
       Paths.get("src/test/resources/projects/Standard8Project2EnvironmentVariables");
   private final File java8Service2EnvVars = pathToJava8Service2WithEnvVars.toFile();
 
-  private final Map<String, String> environment = Maps.newHashMap();
+  // Environment variables included in running the dev server for Java 7/8 runtimes.
```

Comment:
FYI, PRs are more reliable when tests are changed or added *before* the model code is changed. That is, first create a failing test. Then write only as much code as needed to make it pass.